salt/environment_settings.yml

```diff
@@ -602,6 +643,23 @@ environments:
       - aws
     purposes:
       - bootcamps
+    backends:
+      rds:
+        - name: bootcamps
+          multi_az: True
+          engine: postgres
+          vault_plugin: postgresql-database-plugin
+          db_instance_class: db.t2.micro
+          allocated_storage: 50
+          public_access: True
+          engine_version: '9.6.11'
+          purpose: bootcamps
+          mount_point: postgresql-bootcamps
+          parameters:
+            <<: *rds_postgres_9_parameter_group_defaults
+            rds.force_ssl: 1
```

Comment:
This should be a default parameter as it is useful even inside a VPC.

Reply:
I've added this to the PostgreSQL defaults in ff310fb.